bulk-model-sync-gradle-test/src/test/kotlin/org/modelix/model/sync/bulk/gradle/test/PushTest.kt

```diff
@@ -22,34 +28,55 @@ class PushTest {
     private val branchRef = ModelFacade.createBranchReference(RepositoryId("ci-test"), "master")
     private val client = ModelClientV2PlatformSpecificBuilder().url(url).build().apply { runBlocking { init() } }
 
+    private fun runTest(body: (IBranch) -> Unit) {
+        val replicatedModel = client.getReplicatedModel(branchRef)
```

Comment:
`ReplicatedModel` is not needed here, because we only want to read the data once.
And to not call `branch.runRead` you can use `branch.withAutoTransactions()`.

```
val client: IModelClientV2 = TODO()
val branchRef: BranchReference = TODO()
val baseVersion = client.pullIfExists(branchRef)!!
val branch = PBranch(baseVersion.getTree(), client.getIdGenerator()).withAutoTransactions()
```

Reply:
Done. Thanks, I was unaware, that we had AutoTransactions.